src/Plugin_Command.php

```diff
@@ -1195,9 +1225,15 @@ private function get_all_plugins() {
 	 *
 	 * @param array $args Passed-in arguments.
 	 * @param bool $all All flag.
+	 * @param string $exclude Comma separated list of plugin slugs.
```

Comment:
`$verb` param seems to be missing here. Not a problem of this PR per se, but makes sense to add it while we're working on this.
```suggestion
	 * @param string $verb    Optional. Verb to use for the messages. Defaults to 'install'.
	 * @param string $exclude Comma separated list of plugin slugs.
```